spec/libsass-todo-issues/issue_2082/error

```diff
@@ -0,0 +1,10 @@
+DEPRECATION WARNING on line 3 of /sass/spec/libsass-issues/issue_2082/input.scss: #{} interpolation near operators will be simplified
+in a future version of Sass. To preserve the current behavior, use quotes:
+
+  unquote('#{$prefix}"-"#{$suffix}')
+
+You can use the sass-convert command to automatically fix most cases.
+
+Error: Invalid CSS after "": expected selector, was """-bar"
```

Comment:
Sass spec is meant to handle this. Something is up, I need to look into it. This is a low priority issue for now.